### ISSUE TITLE
SC2188 - still a bug (related to #1413)

### ISSUE THREAD

**Issue by notjames** [koalaman/shellcheck#1631]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2188
- My shellcheck version (`shellcheck --version` or "online"): 0.5.0 and 0.6.0
- [X] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [ ] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [X] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash
# shellcheck disable=SC2046
_DIR=$(dirname $(cd -p "$PWD")/..)
TMPL_PATH="$_DIR"/templates
PATH_OF_TRUTH="$_DIR"/../docs/

CONTAINER_ASSETS_PATH=./
CONTAINER_WS_ROOT=/

if [[ -f "$CONTAINER_ASSETS_PATH"/nginx.conf ]]; then
  # shellcheck disable=SC2188
  > "$CONTAINER_ASSETS_PATH"/nginx.conf
fi

```

#### Here's what shellcheck currently says:
1. No errors in nvim; it just blocks permanently (for instance on :w) until interrupted
1. In shell, it returns no output and blocks until interrupted.

#### Here's what I wanted or expected to see:
Something or at least no shellcheck errors.

**Comment by koalaman** [comment]
You're saying you can reproduce this on shellcheck.net or the latest commit? I see the issue with 0.6.0, but I don't see it on either of those.

**Comment by notjames** [comment]
I haven't tried reproducing this issue on shellcheck.net. I've only been running this in CLI and both of the versions mentioned in the issue are affected. I haven't tested this with the latest non-released commit.

**Comment by koalaman** [comment]
Ok. This issue was fixed in #1413, and there's not been a stable release since then so any release versions will still be affected.
